api/src/backend/api/v1/serializers.py

```diff
@@ -2836,3 +2841,275 @@ class Meta:
             "inserted_at": {"read_only": True},
             "last_used_at": {"read_only": True},
         }
+
+
+# Lighthouse: Provider configurations
+
+
+class LighthouseProviderConfigSerializer(RLSSerializer):
```

Comment:
This apply to all serializers, we usually perform the input validation before passing to model. I think that we should the same here, as other serializers do. Check examples like: IntegrationCreateSerializer